extension-readme.md

```diff
@@ -1,182 +1,242 @@
 # <a id="top"></a>AWS Toolkit for Visual Studio Code
 
-The AWS Toolkit for Visual Studio Code is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within Visual Studio Code.
+The *AWS Toolkit for Visual Studio Code* is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within the Visual Studio Code editor.
 
-*This is pre-release software and we recommend against using it in a production environment.*
+The following screenshots show important parts of the Toolkit.
+
+### <a id="ui-components"></a>Fundamental UI Components
 
 ![Overview](./resources/marketplace/overview.png)
```

Comment:
I'm not sure if we need to show the location of the command palette, especially with the gif below.

Reply:
Sure, we can update the screenshot if you think it's distracting.
I was just thinking of that as one of the major parts of the UI for the Toolkit, so wanted to allude to it early and then get more specific.
(I've seen some people have trouble finding it, so wanted to be sure to make it a first-class citizen.)

Reply:
(Done. See below in related comment.)

Reply:
Still seeing this image in the current revision. Is this going to be changed?

Reply:
Yes, the still screenshot has been updated.
If you view the MD file or the PNG file, you will see the change.

Reply:
Weird, might be due to the fact that you're on a fork instead of a branch. Manually opened the file and it looks right, I'll double-check once this is merged to make sure it all looks fine.